sql/core/src/main/scala/org/apache/spark/sql/execution/BaseScriptTransformationExec.scala

```diff
@@ -364,3 +430,46 @@ object ScriptTransformationIOSchema {
       input.schemaLess)
   }
 }
+
+/**
+ * Maps a relative pathname to an absolute pathname using the PATH environment.
+ */
+class PathFinder(val envPath: String) {
```

Comment:
Why do we need this class? We cannot create a helper private method instead?

Reply:
> Why do we need this class? We cannot create a helper private method instead?

Yea, we can. we don't need this after @HyukjinKwon 's idea.